<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientView</class>
 <widget class="QWidget" name="PatientView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PatientView</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,7,3">
     <item>
      <layout class="QFormLayout" name="patientOperationsFormLayout">
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item row="0" column="1">
        <widget class="QGroupBox" name="patientOperationsGroupBox">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>160</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>180</height>
          </size>
         </property>
         <property name="title">
          <string>Patient</string>
         </property>
         <widget class="QLabel" name="patientNameLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
         <widget class="QLabel" name="patientSurnameLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Surname</string>
          </property>
         </widget>
         <widget class="QPushButton" name="addPatientButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>100</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
         <widget class="QPushButton" name="removePatientButton">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>100</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="patientNameLineEdit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>30</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="patientSurnameLineEdit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>60</y>
            <width>101</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="zoneGroupBox">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>190</height>
          </size>
         </property>
         <property name="title">
          <string>Patient's Location</string>
         </property>
         <widget class="QLabel" name="zone_zoneLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Resting at</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="zone_zoneTextEdit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>60</y>
            <width>100</width>
            <height>21</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
          </property>
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QDateEdit" name="zone_inDateEdit">
          <property name="geometry">
           <rect>
            <x>109</x>
            <y>90</y>
            <width>81</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="zone_InDateLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>90</y>
            <width>51</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Arrival</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="zone_outDateEdit">
          <property name="geometry">
           <rect>
            <x>109</x>
            <y>120</y>
            <width>81</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="zone_outDateLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>120</y>
            <width>51</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Departure</string>
          </property>
         </widget>
         <widget class="QPushButton" name="changePatientZoneInfoButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>150</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="zone_inDateCheckBox">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>90</y>
            <width>16</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QCheckBox" name="zone_outDateCheckBox">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>120</y>
            <width>16</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="zone_patientLabel">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Patient</string>
          </property>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>30</y>
            <width>93</width>
            <height>20</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="patientFullNameHorizontalLayout">
           <property name="spacing">
            <number>4</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetFixedSize</enum>
           </property>
           <item>
            <widget class="QLabel" name="zone_patientNameLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>name</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="margin">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="zone_patientSurnameLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>surname</string>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="patientListTableLayout">
       <item>
        <widget class="QGroupBox" name="patientListGroupBox">
         <property name="minimumSize">
          <size>
           <width>530</width>
           <height>300</height>
          </size>
         </property>
         <property name="title">
          <string>Patient List</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="patientListVerticalLayout">
            <item>
             <widget class="QLineEdit" name="searchLineEdit">
              <property name="placeholderText">
               <string>Start typing for searching by name...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="patientsTableView">
              <property name="selectionMode">
               <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="patientMedicinesHorizontalLayout">
       <item>
        <widget class="QGroupBox" name="patientMedicinesGroupBox">
         <property name="title">
          <string>Drugs Used by the Patient</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="patientMedicinesVerticalLayout">
            <item>
             <widget class="QTableView" name="patientMedicinesTableView">
              <property name="selectionMode">
               <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>false</bool>
              </property>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editMedicinesButton">
              <property name="text">
               <string>Edit Used Drugs</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>patientsTableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>zoneGroupBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patientsTableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>editMedicinesButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>774</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>846</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
